include/forcing/Forcing.h

```diff
@@ -133,13 +144,89 @@ class Forcing
 
         struct tm *current_date_time;
 
+        current_date_time_epoch = time_epoch_vector.at(forcing_vector_index_ptr);
+
+        /// \todo: Sort out using local versus UTC time
         current_date_time = localtime(&current_date_time_epoch);
 
         current_day_of_year = current_date_time->tm_yday;
 
         return current_day_of_year;
     }
 
+    /**
```

Comment:
All of these accessors need to share similar bound checking logic on the index.

Reply:
I will put that in.

Reply:
Its probably worth writing a function that does the bound checking and calling it from each accessor.